tutorials/iot-ds-2-create-flp/iot-ds-2-create-flp.md

```diff
@@ -4,7 +4,7 @@ description: Create and configure an SAP Fiori launchpad site to receive notific
 auto_validation: true
 time: 40
 primary_tag: topic>internet-of-things
-tags: [ tutorial>beginner, tutorial>license, topic>internet-of-things, topic>cloud, products>sap-leonardo-iot, products>sap-edge-services, products>sap-cloud-platform-internet-of-things, products>sap-cloud-platform, products>sap-cloud-platform-portal, products>sap-web-ide ]
+tags: [ tutorial>beginner, tutorial>license, topic>internet-of-things, topic>cloud, products>sap-edge-services, products>sap-cloud-platform-internet-of-things, products>sap-business-technology-platform, products>sap-web-ide ]
```

Comment:
products>sap-cloud-platform-internet-of-things -> wrong tag

Reply:
@MichaelCzcz I have removed the wrong tags and replaced the usage of "we/our" with you/your.  I have also made a change regarding the usage of "we" in the rules.vr file for one of the tutorials.  The change was pushed to QA but some how it didn't get pushed to Production: https://github.com/SAPDocuments/Tutorials-Contribution/commit/382239f577ed57556195f8135540a156cca8bcd9. Can I create a separate PR for this change?

Reply:
rules.vr is only pushed to QA, but changes will also work in production.